tests/e2e/Services/VCS/VCSConsoleClientTest.php

```diff
@@ -0,0 +1,316 @@
+<?php
+
+namespace Tests\E2E\Services\VCS;
+
+use Tests\E2E\Scopes\Scope;
+use Tests\E2E\Client;
+use Tests\E2E\Scopes\ProjectCustom;
+use Tests\E2E\Scopes\SideConsole;
+use Utopia\App;
+use Utopia\Database\Helpers\ID;
+use Utopia\Database\Helpers\Role;
+use Utopia\VCS\Adapter\Git\GitHub;
+use Utopia\Cache\Adapter\None;
+use Utopia\Cache\Cache;
+
+class VCSConsoleClientTest extends Scope
+{
+    use ProjectCustom;
+    use SideConsole;
+
+    public ?string $providerRepositoryId = null;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        $this->providerRepositoryId = '705764267';
```

Comment:
If `providerRepositoryId` is hard-coded, we might not need `_APP_VCS_TEST_GITHUB_INSTALLATION_ID` env variable, and simply do variable `providerInstallationId`.
I would prefer having both as ENV variable, but if we want to avoid env vars, we can avoid it for both.